src/main/java/org/jalasoft/moi/model/csharpcompiler/CsharpCompliler.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2020 Jalasoft.
+ *
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Jalasoft.
+ */
+package org.jalasoft.moi.model.csharpcompiler;
+
+import java.io.File;
+import java.io.IOException;
+
+public class CsharpCompliler {
+
+    String cSharpPath;
+
+    public CsharpCompliler() {
+        cSharpPath = "C:\\Windows\\Microsoft.NET\\Framework64\\v4.0.30319\\csc.exe";
+    }
+
+    public File getFile() {
+        String sRuote = "C:\\Users\\Admin\\IdeaProjects\\com\\csharp\\hiworld.cs";
+        File route = new File(sRuote);
+        return route;
+    }
+
+    public String printComplilePath() {
+        if (getFile().exists()) {
+            String fileName = getFile().getName();
+            return cSharpPath + " " + fileName;
+        } else {
+            return "El fichero no existe";
+        }
```

Comment:
I think that this method could be private, and if you need to check what path is created you can use debug tools.